Assets/Scripts/Quest/QuestMonster.cs

```diff
@@ -46,8 +49,8 @@ public QuestMonster(QuestData.UniqueMonster qm) : base()
         }
 
         // Read info from quest data or base type
-        info = EventManager.SymbolReplace(qm.info);
```

Comment:
I think the only place to make symbol replace is from a visual component (textBox) to internal data. After this point, there is no need to replace again because FFG font has all this symbols inside.
After translation, my first idea is to create a textbox rich editor with buttons to insert these special ymbols directly without transformations. and a transform button to make all transformations and look how it will look like before saving.